dataikuapi/dss_plugin_mlflow/utils.py

```diff
@@ -37,6 +37,14 @@ def __init__(self, client, project_key, managed_folder="mlflow_artifacts", host=
             )
         sys.path.insert(0, self.tempdir)
 
+        # Reload the artifact_repository_registry in case MLflow was imported beforehand
+        import mlflow
+        try:  # python 3.4+
+            from importlib import reload
+        except ImportError:
+            pass
+        reload(mlflow.store.artifact.artifact_repository_registry)
```

Comment:
Isn't there a "reload" or reset stuff in mlflow?
Anyway, in the dss packages, we prefer using `sys.version_info` instead of checking `ImportError` (although both works)

Reply:
👍 I have not seen any reload but I am not surprised since normally plugins are installed while installing the package with pip.
I will use sys.versio_info to check on the python version.